### PR TITLE
adding backup and restore file on it's own

### PR DESCRIPTION
**Pull request by cmastr**
Signed-off-by: Casie Oxford <coxford@d2iq.com>

NOTE: You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4110.s3-website-us-west-2.amazonaws.com/

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-78903 

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by gflorinskaya** [comment]
On the page http://docs-d2iq-com-pr-4110.s3-website-us-west-2.amazonaws.com/dkp/kommander/2.1/backup-and-restore/
This link does not work "**Kommander stores all data as CRDs in the Kubernetes API and you can back up and restore it using the [standard Konvoy documentation]**(http://docs-d2iq-com-pr-4110.s3-website-us-west-2.amazonaws.com/dkp/konvoy/latest/backup/)."

**Review by JohnB64-JBV — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.1/backup-and-restore/index.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        # cloud: |
+        #   [default]
+        #   aws_access_key_id=<REDACTED>
+        #   aws_secret_access_key=<REDACTED>
+        ```
+        
+## Install the Velero command-line interface
+
+Although installing the Velero command-line interface is optional and independent of deploying a DKP cluster, having access to the command-line interface provides several benefits.
+For example, you can use the Velero command-line interface to back up or restore a cluster on demand, or to modify certain settings without changing the Velero configuration.
```
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/dkp/kommander/2.1/backup-and-restore/index.md:80]: done

File: pages/dkp/kommander/2.1/backup-and-restore/index.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+By default, Konvoy sets up Velero to use MinIO over TLS using a self-signed certificate.
+As a result, when using certain commands, you may be asked to use the `--insecure-skip-tls-verify` flag.
+Again, the default setup is not suitable for production use-cases.
+
+See the instructions to [install the Velero command-line interface][velero-cli-install] for more information.
+
+In DKP, the Velero platform application is installed in the `kommander` namespace, instead of `velero`.  Thus, after installing the CLI, we recommend that you set the Velero CLI `namespace` config option so that subsequent Velero CLI invocations will use the correct namespace:
```
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/dkp/kommander/2.1/backup-and-restore/index.md:88]: done

**Review by JohnB64-JBV — APPROVED** [review]

**Comment by cmastr** [comment]
> On the page http://docs-d2iq-com-pr-4110.s3-website-us-west-2.amazonaws.com/dkp/kommander/2.1/backup-and-restore/ This link does not work "**Kommander stores all data as CRDs in the Kubernetes API and you can back up and restore it using the [standard Konvoy documentation]**(http://docs-d2iq-com-pr-4110.s3-website-us-west-2.amazonaws.com/dkp/konvoy/latest/backup/)."

This is fixed. I just took out the link bc it's now this topic.

File: pages/dkp/kommander/2.1/backup-and-restore/index.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+  name: velero-overrides
+  namespace: kommander
+data:
+  values.yaml: |
+     minio:
+      persistence:
+         storageClass: <external storage class name>
+         ```
+
+You can also store your backups in Amazon S3 by configuring Velero in `cluster.yaml`  as follows:
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/backup-and-restore/index.md:38]: take out?

**Pull request merged**